[2024-04-18T19:03:43.958+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Units_statistics_DAG.split_data_into_chunks manual__2024-04-18T19:03:35.196117+00:00 [queued]>
[2024-04-18T19:03:43.972+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Units_statistics_DAG.split_data_into_chunks manual__2024-04-18T19:03:35.196117+00:00 [queued]>
[2024-04-18T19:03:43.973+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 4
[2024-04-18T19:03:43.994+0000] {taskinstance.py:2217} INFO - Executing <Task(_PythonDecoratedOperator): split_data_into_chunks> on 2024-04-18 19:03:35.196117+00:00
[2024-04-18T19:03:44.004+0000] {standard_task_runner.py:60} INFO - Started process 386700 to run task
[2024-04-18T19:03:44.012+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'Units_statistics_DAG', 'split_data_into_chunks', 'manual__2024-04-18T19:03:35.196117+00:00', '--job-id', '7822', '--raw', '--subdir', 'DAGS_FOLDER/Units_dag.py', '--cfg-path', '/tmp/tmp35az2re7']
[2024-04-18T19:03:44.014+0000] {standard_task_runner.py:88} INFO - Job 7822: Subtask split_data_into_chunks
[2024-04-18T19:03:44.106+0000] {task_command.py:423} INFO - Running <TaskInstance: Units_statistics_DAG.split_data_into_chunks manual__2024-04-18T19:03:35.196117+00:00 [running]> on host a09da322e808
[2024-04-18T19:03:44.240+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='abeer-araby' AIRFLOW_CTX_DAG_ID='Units_statistics_DAG' AIRFLOW_CTX_TASK_ID='split_data_into_chunks' AIRFLOW_CTX_EXECUTION_DATE='2024-04-18T19:03:35.196117+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-18T19:03:35.196117+00:00'
[2024-04-18T19:03:44.263+0000] {Units_dag.py:78} INFO - type of length is :<class 'int'>
[2024-04-18T19:03:44.785+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/elasticsearch/_sync/client/__init__.py:399: SecurityWarning: Connecting to 'https://dbelk3.afaqy.local:9200' using TLS with verify_certs=False is insecure
  _transport = transport_class(

[2024-04-18T19:03:44.833+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/elasticsearch/_sync/client/__init__.py:399: SecurityWarning: Connecting to 'https://dbelk2.afaqy.local:9200' using TLS with verify_certs=False is insecure
  _transport = transport_class(

[2024-04-18T19:03:44.881+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/elasticsearch/_sync/client/__init__.py:399: SecurityWarning: Connecting to 'https://dbelk1.afaqy.local:9200' using TLS with verify_certs=False is insecure
  _transport = transport_class(

[2024-04-18T19:03:44.905+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/urllib3/connectionpool.py:1061: InsecureRequestWarning: Unverified HTTPS request is being made to host 'dbelk3.afaqy.local'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(

[2024-04-18T19:03:44.971+0000] {_transport.py:335} INFO - GET https://dbelk3.afaqy.local:9200/_cluster/health [status:200 duration:0.087s]
[2024-04-18T19:03:45.010+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/urllib3/connectionpool.py:1061: InsecureRequestWarning: Unverified HTTPS request is being made to host 'dbelk2.afaqy.local'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(

[2024-04-18T19:03:45.015+0000] {_transport.py:335} INFO - HEAD https://dbelk2.afaqy.local:9200/units_statistics_*** [status:200 duration:0.042s]
[2024-04-18T19:03:45.046+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/urllib3/connectionpool.py:1061: InsecureRequestWarning: Unverified HTTPS request is being made to host 'dbelk1.afaqy.local'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(

[2024-04-18T19:03:45.393+0000] {_transport.py:335} INFO - DELETE https://dbelk1.afaqy.local:9200/units_statistics_*** [status:200 duration:0.377s]
[2024-04-18T19:03:45.395+0000] {Units_dag.py:91} INFO - i = 0, chunk = 10000
[2024-04-18T19:03:45.396+0000] {Units_dag.py:91} INFO - i = 10000, chunk = 10000
[2024-04-18T19:03:45.397+0000] {Units_dag.py:91} INFO - i = 20000, chunk = 10000
[2024-04-18T19:03:45.397+0000] {Units_dag.py:91} INFO - i = 30000, chunk = 10000
[2024-04-18T19:03:45.398+0000] {Units_dag.py:91} INFO - i = 40000, chunk = 10000
[2024-04-18T19:03:45.398+0000] {Units_dag.py:91} INFO - i = 50000, chunk = 10000
[2024-04-18T19:03:45.398+0000] {Units_dag.py:91} INFO - i = 60000, chunk = 10000
[2024-04-18T19:03:45.399+0000] {Units_dag.py:91} INFO - i = 70000, chunk = 10000
[2024-04-18T19:03:45.399+0000] {Units_dag.py:91} INFO - i = 80000, chunk = 10000
[2024-04-18T19:03:45.400+0000] {Units_dag.py:91} INFO - i = 90000, chunk = 10000
[2024-04-18T19:03:45.400+0000] {Units_dag.py:91} INFO - i = 100000, chunk = 10000
[2024-04-18T19:03:45.400+0000] {Units_dag.py:91} INFO - i = 110000, chunk = 10000
[2024-04-18T19:03:45.401+0000] {Units_dag.py:91} INFO - i = 120000, chunk = 10000
[2024-04-18T19:03:45.401+0000] {Units_dag.py:91} INFO - i = 130000, chunk = 10000
[2024-04-18T19:03:45.401+0000] {Units_dag.py:91} INFO - i = 140000, chunk = 10000
[2024-04-18T19:03:45.401+0000] {Units_dag.py:91} INFO - i = 150000, chunk = 10000
[2024-04-18T19:03:45.401+0000] {Units_dag.py:91} INFO - i = 160000, chunk = 6595
[2024-04-18T19:03:45.425+0000] {python.py:202} INFO - Done. Returned value was: [[0, 10000], [10000, 10000], [20000, 10000], [30000, 10000], [40000, 10000], [50000, 10000], [60000, 10000], [70000, 10000], [80000, 10000], [90000, 10000], [100000, 10000], [110000, 10000], [120000, 10000], [130000, 10000], [140000, 10000], [150000, 10000], [160000, 6595]]
[2024-04-18T19:03:45.466+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=Units_statistics_DAG, task_id=split_data_into_chunks, execution_date=20240418T190335, start_date=20240418T190343, end_date=20240418T190345
[2024-04-18T19:03:45.511+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-18T19:03:45.671+0000] {taskinstance.py:3312} INFO - 17 downstream tasks scheduled from follow-on schedule check
